### ISSUE TITLE
How could build asmc in Linux now?

### ISSUE THREAD

**Issue by zyppe** [nidud/asmc#40]
I amd trying build asmc in my linux but I don't know how to do it.
I see it seems to be a file called GccLinux.mak but I cannot find where is it. So I use "cd source/asmc/gcc && make", But it failed.

**Comment by teoberi** [comment]
Where did you find the GccLinux.mak file?

I compiled version 2.34.34 a few seconds ago without any errors.
Post the lines with the error and the version of GCC.

**Comment by nidud** [comment]
I added some new features lately, among other tings the directive LDR to load arguments based on calling conventions. This enables more portable code for 32/64 bit Linux and Windows, which is now applied to the Asmc source.

However, I didn't update the Linux binaries so you get:
```
error A2008: syntax error : ldr
```
There's another general issue in Linux where some distributions are exclusively 64-bit and others fails using `asmc64`, assuming based on the library used building Asmc (latest version of Ubuntu in this case). For this reason a 32-bit static version is added. If this fails change the first line in the `gcc/makefile`:
```
acc = asmc64
```
to
```
acc = asmc
```
Well, I updated the Linux binaries so the LDR issue should now be solved.
